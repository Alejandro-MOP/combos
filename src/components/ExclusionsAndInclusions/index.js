export { default as GeneralDirectorateSales } from './GeneralDirectorateSales/GeneralDirectorateSales';
export { default as SalesOrganization } from './SalesOrganization/SalesOrganization';
export { default as SalesOffice } from './SalesOfice/SalesOffice';
export { default as Customers } from './Customers/Customers';
