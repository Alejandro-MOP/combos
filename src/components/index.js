export { default as Head } from './Head/Head'
export { default as Nav } from './Nav/Nav'
export { default as LimiteCombos } from './LimiteCombos/LimiteCombos'