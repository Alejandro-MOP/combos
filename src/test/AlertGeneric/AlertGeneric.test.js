import '@testing-library/jest-dom'
import React from 'react';
import { shallow } from 'enzyme'

import { AlertGeneric } from '../../components';

describe('Test in <AlertGeneric />', () => {
    const severity = 'warning';
    const text = 'text for test';
    const wrapper = shallow(
        <AlertGeneric
            text={text}
            severity={severity}
        />
    );

    test('should show <AlertGeneric /> correctly', () => {

        expect(wrapper).toMatchSnapshot();
    });

});