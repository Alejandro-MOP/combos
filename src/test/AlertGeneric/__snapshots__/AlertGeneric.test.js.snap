// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <AlertGeneric /> should show <AlertGeneric /> correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Alert))
    severity="warning"
  >
    text for test
  </WithStyles(ForwardRef(Alert))>
</div>
`;
